addons/wowl/static/tests/mock_server.ts

```diff
@@ -0,0 +1,107 @@
+import { Component } from "@odoo/owl";
+import { OdooEnv, Service } from '../src/types';
+import { Model, ModelBuilder , DBRecord } from '../src/services/model';
+import { Action } from '../src/services/action_manager/helpers';
+
+// Aims:
+// - Mock service model high level
+// - propose mock model.call lower level
+// - propose mock RPC low level
+
+// Can be passed data
+// returns at least model service
+
+type FieldType =
```

Comment:
I think that you can move these types in the main types.ts file. We will need them soon when we work on fields view get and stuff like that